specification.md

```diff
@@ -3144,14 +3144,14 @@ reduce the number of connections to the Server when a very large number
 
 ### Configuration Management
 
-* [Uber Flipr](https://eng.uber.com/flipr/)
+* [Uber Flipr](https://web.archive.org/web/20240223111552/https://www.uber.com/blog/flipr/)
 * Facebook's
   [Holistic Configuration Management](https://research.facebook.com/file/877841159827226/holistic-configuration-management-at-facebook.pdf)
   (push)
 
 ### Security and Certificate Management
 
-* [mTLS in Go](https://kofo.dev/how-to-mtls-in-golang)
+* [mTLS in Go](https://web.archive.org/web/20240223142649/https://kofo.dev/how-to-mtls-in-golang)
```

Comment:
This person's website has gone offline so I replaced this with the link from the Wayback Machine.